class.jetpack-user-agent.php

```diff
@@ -695,6 +695,28 @@ static function is_firefox_mobile( ) {
 			return false;
 	}
 
+	/*
+	 * Detects if the current browser is Firefox for desktop
+	 *
+	 * https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox
+	 * Mozilla/5.0 (platform; rv:geckoversion) Gecko/geckotrail Firefox/firefoxversion
+	 * The platform section will include 'Mobile' for phones and 'Tablet' for tablets.
+	 *
+	 */
+	static function is_firefox_desktop() {
+
+		if ( empty( $_SERVER['HTTP_USER_AGENT'] ) ) {
+			return false;
+		}
+
+		$ua = strtolower( $_SERVER['HTTP_USER_AGENT'] );
+
+		if ( false !== strpos( $ua, 'firefox' ) && false === strpos( $ua, 'mobile' ) && false === strpos( $ua, 'tablet' ) ) {
```

Comment:
Since the `strpos()` function is case-sensitive, maybe we should say `Mobile` and `Tablet`?

Reply:
`$ua` is the user agent string converted to lowercase (see line 712). So, I think that `mobile` and `tablet` are the correct strings to use.

Reply:
Oh yeah, my bad!